#20804 add a note for unique per instance extract directory

2m opened the pull request:
If extract directory for sigar loader is shared, first instance
JVM crashes when the second one is started.

Fixes #20804

Andrei-Pozolotin commented on akka-docs/rst/java/cluster-metrics.rst:
```diff
@@ -90,8 +90,15 @@ User can provision Sigar classes and native library in one of the following ways
+
+  When using `Kamon sigar-loader <https://github.com/kamon-io/sigar-loader>`_ and running multiple
+  instances of the same application, you have to make sure that sigar library is extracted to a
+  unique per instance directory. You can control theextract directory with the
```
missing space "the extract"

Andrei-Pozolotin commented on akka-docs/rst/scala/cluster-metrics.rst:
```diff
@@ -86,8 +86,15 @@ User can provision Sigar classes and native library in one of the following ways
+
+  When using `Kamon sigar-loader <https://github.com/kamon-io/sigar-loader>`_ and running multiple
+  instances of the same application, you have to make sure that sigar library is extracted to a
+  unique per instance directory. You can control theextract directory with the
```
missing space "the extract"

Andrei-Pozolotin commented on akka-docs/rst/java/cluster-metrics.rst:
```diff
@@ -90,8 +90,15 @@ User can provision Sigar classes and native library in one of the following ways
+.. warning::
+
+  When using `Kamon sigar-loader <https://github.com/kamon-io/sigar-loader>`_ and running multiple
+  instances of the same application, you have to make sure that sigar library is extracted to a
```
say "the same application on the same host"

Andrei-Pozolotin commented on akka-docs/rst/scala/cluster-metrics.rst:
```diff
@@ -86,8 +86,15 @@ User can provision Sigar classes and native library in one of the following ways
+.. warning::
+
+  When using `Kamon sigar-loader <https://github.com/kamon-io/sigar-loader>`_ and running multiple
+  instances of the same application, you have to make sure that sigar library is extracted to a
```
say "the same application on the same host"

ktoso commented:
LGTM after the 2 comments by @Andrei-Pozolotin are addressed.

2m committed:
#20804 add a note for unique per instance extract directory

If extract directory for sigar loader is shared, first instance
JVM crashes when the second one is started.
